### PR TITLE
Honour Markdown in lists

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony @saeedmb8 ready for review. Do we usually write tests for templatetags?

File: ndohyep/templates/core/article_page.html [hunk 1]
```diff
@@ -40,14 +40,22 @@
       {% if block.block_type == 'numbered_list' %}
         <ol>
         {% for item in block.value %}
-          <li>{{item}}</li>
+          <li>{{ item|handle_markdown }}</li>
+        {% endfor %}
+        </ol>
+      {% else %}
```
Comment (hodgestar) [inline ndohyep/templates/core/article_page.html]: You can use a `{% elif ... %}` here.

**Comment by hodgestar** [comment]
We do usually try to test templatetags.

**Comment by miltontony** [comment]
You can test using [this](https://github.com/praekelt/molo/blob/develop/molo/core/tests/test_views.py#L198-L210)

File: ndohyep/templatetags/ndohyep_tags.py [hunk 2]
```diff
@@ -16,4 +18,15 @@ def is_in_group(user, group_name):
...
 
+def handle_markdown(value):
+    md = markdown(
+        value,
+        [
+            'markdown.extensions.fenced_code',
+            'codehilite',
+        ]
+    )
+    return mark_safe(md)
```
Comment (KaitCrawford) [inline ndohyep/templatetags/ndohyep_tags.py:36]: @miltontony there's a slight problem in that this code is adding `<p></p>` tags around each list item. Any idea how we can stop it doing that?

**Comment by KaitCrawford** [comment]
@miltontony @hodgestar @saeedmb8 ready for re-review

**Comment by KaitCrawford** [comment]
@miltontony since you gave me a :+1: on the PR in molo (which was almost a duplicate of this) are you happy to do the same here?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**